src/icons/file/README.md

```diff
@@ -21,111 +21,107 @@ const IconsExample = require('../../../examples/src/IconsExample').default;
 const icons = [
     {
         name: 'IconFileAudio',
-        component: require('./IconFileAudio').default,
+        component: require('../imported/Icon/IconFileAudio32').default,
     },
     {
         name: 'IconFileBoxNote',
-        component: require('./IconFileBoxNote').default,
+        component: require('../imported/Icon/IconFileBoxNote32').default,
     },
     {
         name: 'IconFileCode',
-        component: require('./IconFileCode').default,
+        component: require('../imported/Icon/IconFileCode32').default,
     },
     {
         name: 'IconFileDefault',
-        component: require('./IconFileDefault').default,
-    },
-    {
-        name: 'IconFileDocument',
-        component: require('./IconFileDocument').default,
+        component: require('../imported/Icon/IconFileDefault32').default,
     },
     {
         name: 'IconFileDwg',
-        component: require('./IconFileDwg').default,
+        component: require('../imported/Icon/IconFIleDwg32').default,
```

Comment:
Typo?

Reply:
the originating file had this in, so had to push an update then re-import. should be fixed now